[WIP] Consolidated 4.10 RAN docs (#43890)
openshift/openshift-docs: modules/about-ztp-and-distributed-units-on-openshift-clusters.adoc

// Module included in the following assemblies:
//
// *scalability_and_performance/ztp-deploying-disconnected.adoc

:_content-type: CONCEPT
[id="about-ztp-and-distributed-units-on-openshift-clusters_{context}"]
= About ZTP and distributed units on OpenShift clusters

> You can install a distributed unit (DU) on a single node cluster at scale with {rh-rhacm-first} using the assisted installer (AI) and the policy generator with core-reduction technology enabled. The DU installation is done using zero touch provisioning (ZTP) in a disconnected environment.

Reviewer: ```suggestion
You can install a distributed unit (DU) on a {product-title} clusters at scale with {rh-rhacm-first} using the assisted installer (AI) and the policy generator with core-reduction technology enabled. The DU installation is done using zero touch provisioning (ZTP) in a disconnected environment.
```

> {rh-rhacm} manages clusters in a hub-and-spoke architecture, where a single hub cluster manages many spoke clusters. {rh-rhacm} applies radio access network (RAN) policies from predefined custom resources (CRs). Hub clusters running ACM provision and deploy the spoke clusters using ZTP and AI. DU installation follows the AI installation of {product-title} on a single node.

Reviewer: ```suggestion
{rh-rhacm} manages clusters in a hub-and-spoke architecture, where a single hub cluster manages many spoke clusters. {rh-rhacm} applies radio access network (RAN) policies from predefined custom resources (CRs). Hub clusters running ACM provision and deploy the spoke clusters using ZTP and AI. DU installation follows the AI installation of {product-title} on each cluster.
```

> The AI service handles provisioning of {product-title} on single nodes running on bare metal. ACM ships with and deploys the AI when the `MultiClusterHub` custom resource is installed.

Reviewer: ```suggestion
The AI service handles provisioning of {product-title} on single nodes, 3-node clusters, or standard clusters running on bare metal. ACM ships with and deploys the AI when the `MultiClusterHub` custom resource is installed.
```
